package.json

```diff
@@ -50,29 +49,33 @@
     "@pollyjs/adapter-node-http": "^5.0.0",
     "@pollyjs/core": "^5.0.0",
     "@pollyjs/persister-fs": "^5.0.0",
+    "@swc/core": "^1.11.11",
+    "@swc/jest": "^0.2.37",
     "@testing-library/dom": "^10.4.0",
     "@testing-library/react": "10.0.0",
+    "babel-jest": "^29.7.0",
     "babel-loader": "^9.2.1",
     "babel-preset-jest": "^29.6.3",
     "codecov": "3.7.2",
+    "cross-fetch": "^4.1.0",
```

Comment:
FWIW in the profiler server, we were using `node-fetch`, this was working fine.
But we could remove it after the upgrade to node v22 which supports fetch natively by default.

Reply:
Yes, node-fetch isn't working some other dependencies apparently. It's a mess.